stable/agent/templates/deployment.yaml

```diff
@@ -81,6 +88,25 @@ spec:
             - name: docker-config
               mountPath: /root/.docker
             {{- end }}
+            {{- if .Values.dind.enabled }}
+            - name: shared-volume
+              mountPath: "/var/buildkite"
```

Comment:
Should this be `/buildkite`? (the [default location for docker](https://buildkite.com/docs/agent/v3/docker#default-file-locations))